README.md

```diff
@@ -1,9 +1,21 @@
 # Eludris
 
-A simple FOSS federated decentralized blazingly fast(:rocket:) social media
-platform with e2ee support made in rust(:rocket:). :rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket:
+A free and open source, federated, End-to-end-Encrypted social media platform made in rust that's easy to deploy and configure while striving to be *truly **yours***.
+
+## Deployment
+
+We really recommend and *only* officially support using the provided docker-compose as a quick way to get stuff running, just edit your `Eludris.toml` to suit your needs then run
+
+```sh
+docker-compose up
+```
+
+Congratulations, you've now successfully deployed your Eludris instance!
```

Comment:
```suggestion
Congratulations, you've now successfully deployed your Eludris instance! 🚀
```

Reply:
Now, this minor change had rather huge consequences on the foreseeable future of Eludris as it drastically changes the way Eludris is marketed and the target audience more likely to use it, I'm not quite sure if that's desirable yet which is why I might open a DEP soon, for now I'll let the other members vote on this change.

TLDR: rocket emoji cringe

Reply:
but is it the Rust way to not use the sacred "🚀"

Reply:
> "Eludris needs trump any Rust needs"
\- Not Linus Torvalds

Reply:
You could not be more wrong. The rocket emoji here does not symoblize Eludris speed or a similarly outrageous statement, it is meant to make the user really feel their accomplisment of finally installing Eludris and switching to a better life.

\- tea :rocket:

Reply:
How about instead of cringe rocket emoji we make a tiny thang emoji with confetti?

Reply:
might actually do that tomorrow when I have time then upload it to tooty effis static when I add that